test/utils/configuration/configuration.go

```diff
@@ -58,14 +61,28 @@ func init() {
 		panic(fmt.Errorf("failed to parse machine config pool update interval: %w", err))
 	}
 
-	Platform, err = detect.Detect()
-	if err != nil {
+	platDetect, reason := detect.FindPlatform(platform.Platform("Unknown"))
```

Comment:
in this case we do want to try autodetecting, so I'd use `detect.Platform()`
additionally, `Platform` is a really bad name for a variable (my fault) so let's take this chance to rename it to anything else which doesn't clash with the `platform.Platform` type.

Reply:
Ack, fixed.